community/php5/APKBUILD

```diff
@@ -438,6 +439,7 @@ _mv_ext() {
 	local ini=$ext.ini
 	pkgdesc="${ext} extension for PHP"
 	provides="php-$extname"
+	depends=
```

Comment:
I think better remove it few lines bellow
```diff
-  depends="${pkgname}-common ${depends}"
+  depends="${pkgname}-common"
```

Reply:
ah.. I'm wrong... it just needs to be
```
depends="${pkgname}-common"
	# extension dependencies
	if [ -n "${2-}" ]; then
 		depends="${2-} ${depends}"
fi
```

Reply:
No, we must clean 'depends' variable first. Then populate it with extension-specific depends (if so) and finally add php5-common (config) dependency

Reply:
Then just make it `${depends}" depends="${2-}` - to append common if any dependencies added